sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ParallelTransferOptions.java

```diff
@@ -0,0 +1,63 @@
+package com.azure.storage.blob.models;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.common.Constants;
+import com.azure.storage.common.Utility;
+
+public class ParallelTransferOptions {
+    private final ClientLogger logger = new ClientLogger(ParallelTransferOptions.class);
+
+    static final int BLOB_DEFAULT_UPLOAD_BLOCK_SIZE = 4 * Constants.MB;
+    static final int BLOB_MAX_BLOCK_SIZE = 100 * Constants.MB;
```

Comment:
I think we're not enforcing service limitations unless there's a really good reason to.

Reply:
Additionally, this maximum value wouldn't apply to a download. Only to put block operations.

Reply:
Actually, in the original download to file code we had a BLOB_MAX_DOWNLOAD_BLOCK_SIZE = 100MB constant that we were checking